tap_helpscout/schemas/happiness_ratings.json

```diff
@@ -0,0 +1,44 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "thread_id": {
+      "type": ["null", "integer"]
+    },
+    "conversation_id": {
+      "type": ["null", "integer"]
```

Comment:
I'm going to remove `null` because I'm 100% confident that these fields can never null from source API response.
I observed for most of the integrations we included `null` for PKs, I feel this is just to avoid unwanted loading errors if by mistake any field comes as null from source API

Examples:

- [Hubspot customer PK](https://github.com/singer-io/tap-hubspot/blob/master/tap_hubspot/schemas/companies.json#L7-L8)
- [Intercom company PK](https://github.com/singer-io/tap-intercom/blob/master/tap_intercom/schemas/companies.json#L16-L17)